package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@random-guys/bucket",
-  "version": "0.10.0",
+  "version": "0.10.1",
```

Comment:
avoid changing version nos in feature requests...plus this a feature si it needs a minor version change, not a patch